packages/types/src/documents/app/automation/StepInputsOutputs.ts

```diff
@@ -100,7 +100,7 @@ export type FilterStepOutputs = BaseAutomationOutputs & {
 export type LoopStepInputs = {
   option: LoopStepType
   binding: any
-  iterations?: number
+  iterations?: string
```

Comment:
Why this? Should this not always be a number and cast any possible usage that comes as a string?

Reply:
Using parseInt on a number gives type errors. And the frontend always sends this back as a string.

Reply:
Maybe there is something else we should handle then. Not sure if the solution is to make our types more dirty

Reply:
Changed the type back to number and made sure we run it through a parser function before use